core/lexicon/en/default.inc.php

```diff
@@ -130,6 +130,7 @@
 $_lang['edit_snippet'] = 'Edit Snippet';
 $_lang['edit_template'] = 'Edit Template';
 $_lang['edit_tv'] = 'Edit TV';
+$_lang['editable_field'] = 'Double-click for quick editing.';
```

Comment:
```suggestion
$_lang['editable_field'] = 'Double-click to open edit mode.';
```

I believe this is a more accurate description for the "double-click" action. Sometimes you are able to edit the value directly, sometimes it shows a select box you have to click if you want to change the value.